README.md

```diff
@@ -1,12 +1,48 @@
+
+## Register an Edge Node with your Mangement Hub
+In order to deploy a service to an edge node it must first be registered with a management hub. The `agent-install.sh` script is a fast and easy way to register an edge node with a management hub, more information can be found in the [open-horizon/anax](https://github.com/open-horizon/anax/tree/master/agent-install#edge-node-agent-install) repository. Edge nodes can be either a device or a cluster. Open Horizon edge cluster capability helps you manage and deploy workloads from a management hub cluster to remote instances of OpenShift® Container Platform or other Kubernetes-based clusters. 
+
+Typically, **edge devices** have a prescriptive purpose, provide (often limited) compute capabilities, and are located near or at the data source. Currently supported edge device OS and architectures:
+* Ubuntu bionic, xenial
+  * arm64, amd64
+* Raspberry Pi OS buster, stretch
+  * armhf
+* Debian buster, stretch
+  * amd64
+* macOS
+  * amd64
+
+Open Horizon **edge cluster** capability helps you manage and deploy workloads from a management hub cluster to remote instances of OpenShift® Container Platform or other Kubernetes-based clusters. Edge clusters are edge nodes that are Kubernetes clusters. An edge cluster enables use cases at the edge, which require colocation of compute with business operations, or that require more scalability, availability, and compute capability than what can be supported by an edge device. Further, it is not uncommon for edge clusters to provide application services that are needed to support services running on edge devices due to their close proximity to edge devices. Open Horizon deploys edge services to an edge cluster, via a Kubernetes operator, enabling the same autonomous deployment mechanisms used with edge devices. The full power of Kubernetes as a container management platform is available for edge services that are deployed by Open Horizon. Currently supported edge cluster architectures: 
+* [OCP on x86_64 platforms](https://docs.openshift.com/container-platform/4.5/welcome/index.html)
+* [K3s - Lightweight Kubernetes](https://rancher.com/docs/k3s/latest/en/)
+* [MicroK8s](https://microk8s.io/docs) on Ubuntu 18.04 (for development and test, not recommended for production)
+
+Currently there is only one example service in this repository that is designed to run on an edge cluster and that is the [nginx-operator](edge/services/nginx-operator).
+
+# Example Services 
+During the management hub installation, several services should have been published into the exchange automatically. The following three command will list the services, patterns, and deployment policies available in your exchange:
+```
+hzn exchange service list IBM/
+hzn exchange pattern list
+hzn exchange deployment listpolicy 
+```
+**Note:** The above commands assume you have `HZN_ORG_ID` set to the created user org (default is `mycluster`), and your `HZN_EXCHANGE_USER_AUTH` set accordingly.
```

Comment:
The org default is sometimes `myorg` as well.

Reply:
Ah, good point! I forget that's the default when running the all-in-1 script. I'll have to add that in